### ISSUE TITLE
Node API

### ISSUE THREAD

**Issue by goodmind** [flow-typed/flow-typed#3122]
It would be cool if you had programmatic API for Node
```js
const {search, install} = require('flow-typed')

function main() {
  const name = await getName()
  const response = await search(name)
  ...
}
```

**Comment by villesau** [comment]
This might actually be surprisingly easy to implement. See the commands: https://github.com/flow-typed/flow-typed/tree/master/cli/src/commands

All of them have identical interface: https://github.com/flow-typed/flow-typed/blob/master/cli/src/commands/runTests.js#L630

What's the use case? I think we could accept pull request that implements programmatic API, but note that flow-typed has pretty complex file system and git dependencies. e.g `.flow-typed` cache folder might be a bit surprising for user of the programmatic API.

**Comment by goodmind** [comment]
@villesau never seen `.flow-typed` folder

**Comment by Brianzchen** [comment]
Merged into `main`. will be available in `4.0.0` release
